app/views/gardens/_area.html.haml

```diff
@@ -0,0 +1,15 @@
+- if garden.area
+  %span.area
+    = "#{garden.area} "
+    - if garden.area_unit === 'square metre'
+      %span{:title => 'square metre'}
+        m
+        %sup 2
+    - elsif garden.area_unit === 'square foot'
+      %span{:title => 'square foot'}
+        ft
+        %sup 2
+    - elsif garden.area_unit === 'hectare'
+      %span{:title => 'hectare'} HA
+    - elsif garden.area_unit === 'acre'
+      %span{:title => 'acre'} A
```

Comment:
You can make a new file in `app/helpers`. You can return an html string from the helper and just call `html_safe` on it to make sure all your brackets aren't escaped.